packages/web/src/scss/components/Pagination/_theme.scss

```diff
@@ -0,0 +1,21 @@
+@use 'sass:map';
+@use '@tokens' as tokens;
+
+$item-gap: tokens.$space-300;
+$item-min-width: 2em;
+$item-height: 2.5em;
+$item-padding-inline: tokens.$space-400;
+$item-emphasis-font-weight: map.get(tokens.$body-medium-text-bold, 'mobile', 'font-weight');
+$item-radius: tokens.$radius-100;
+
+$item-default-color: tokens.$text-secondary-default;
+$item-default-background: transparent;
```

Comment:
I think it should be white 
![image](https://user-images.githubusercontent.com/7041807/191765190-ecb0259d-d265-4d26-827c-aec6ae5026ae.png)

Reply:
Fixed.